cli/azd/pkg/tools/kubectl/models_test.go

```diff
@@ -0,0 +1,34 @@
+package kubectl
+
+import (
+	"encoding/json"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func Test_Port_TargetPort_Unmarshalling(t *testing.T) {
```

Comment:
Happy to see this test to demonstrate the failing case and make the output contact known, but curious if adding a comment may serve the same purpose. I certainly would have a double-take when changing `any` to `int` due to the failing test for example, but the comment would have alluded at that early enough before the change was made.

Alternatively, perhaps a comment that suggests `port: redis` is a well-known real-world test case may help bring concreteness.

Not sure what type of coding styles our team prefers.